src/ast/compare.rs

```diff
@@ -44,6 +44,30 @@ pub enum Compare<'a> {
     /// Raw comparator, allows to use an operator `left <raw> right` as is,
     /// without visitor transformation in between.
     Raw(Box<Expression<'a>>, Cow<'a, str>, Box<Expression<'a>>),
+    #[cfg(all(feature = "json", any(feature = "postgresql", feature = "mysql")))]
```

Comment:
I feel like the JSON ops could have been encapsulated in another enum that in turn is included here. Blows up the compare AST too much for my taste.

Reply:
yeah that looks much better now.